docs/install.md

```diff
@@ -17,6 +17,24 @@ npm install -S emotion
 }
 ```
 
+**webpack.config.js**
+
+If you keep babel-loader configuration in your Webpack config, add Emotion as the first item in the plugins section:
```

Comment:
Why is it important that it's the first item? Why is this specific to Babel config in `webpack.config.js`? Maybe that explanation could be added too? 🤔

Reply:
@silvenon thanks for the review. i have added a separate note about plugin order, since it's not specific to webpack.config.js, and quickly explained why it's significant.